life/modernized/02-chapter-1.adoc

```diff
@@ -23,7 +24,10 @@ He has left a particular account of his early religious experience, in a work en
 "`The Inheritance of Jacob discovered after his return out of Egypt;`"
 from which the reader is here presented with a copious extract.
 
-"`From twelve years old I set my heart to know that God whom the world professed,
+[.embedded-content-document]
```

Comment:
I hemmed and hawed over this for a bit of whether it should be a [quote] instead because it's from a different book, but it's FH speaking, so idk!

Reply:
I think I would have done the same thing👍🏻